node/get-data.js

```diff
@@ -26,8 +26,8 @@ async function main () {
 		if (!pth.endsWith(".json")) continue;
 		if (_BLACKLIST_FILENAMES_JSON.has(path.basename(pth))) continue;
 		const pathSiteDir = path.join(process.argv[2], pth);
-		if (!fs.existsSync(pathSiteDir)) throw new Error(`File ${pth} does not exist in 5etools data!`);
-		fs.copyFileSync(pathSiteDir, pth);
+		if (!fs.existsSync(pathSiteDir)) console.log(`File ${pth} does not exist in 5etools data!`);
```

Comment:
Does this exist? Or should I make it?

Reply:
`_BLACKLIST_FILENAMES_JSON` is on line `10` of this file; instead of logging the bad files, just add the "known bad paths" to the `_BLOCKLIST`. Then we can keep the `throw new Error(...)`, which is marginally more idiot-proof